docs/reference/specifications/custom-operations/semver-operation-spec.md

```diff
@@ -21,7 +21,7 @@ The 'sem_ver' evaluation rule contains exactly three items:
 1. Target property value: the resolved value of the target property referenced in the targeting rule
 2. Operator: One of the following: `=`, `!=`, `>`, `<`, `>=`, `<=`, `~` (match minor version), `^` (match major version)
 3. Target value: this needs to resolve to a semantic versioning string. If this condition is not met, the evaluator should
-log an appropriate error message and return `nil`
+log an appropriate error message and return `false`
```

Comment:
Should we specific the log level?

Reply:
I think that's probably too far, I'm not sure they are the same for all languages. It feels like something devs can make their own call on